sale_invoice_plan/models/sale.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from dateutil.relativedelta import relativedelta
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.addons import decimal_precision as dp
+from odoo.tools.float_utils import float_compare, float_round
+
+
+class SaleOder(models.Model):
```

Comment:
s/SaleOder/SaleOrder

Reply:
@bealdav thanks for reviewing. I am not sure what your comment is about :p

Reply:
oh, it's just a typo error.
You wrote `SaleOder`, I suppose you would write `SaleOrder` :)